pkg/models/feature_flags_model.go

```diff
@@ -67,16 +69,24 @@ type FeatureFlagRecord struct {
 	Name           string             `json:"name" bson:"name"`
 	Type           FlagType           `json:"type" bson:"type"`
 	Revisions      []Revision         `json:"revisions" bson:"revisions"`
+	Timeline       string             `json:"timeline,omitempty" bson:"timeline,omitempty"`
```

Comment:
I don't think timeline is a string

Reply:
This was removed. A PR to add Timeline will be better